src/systems/material.js

```diff
@@ -39,6 +39,9 @@ module.exports.System = registerSystem('material', {
    */
   loadImage: function (el, material, data, src) {
     var repeat = data.repeat || '1 1';
+    var offset = data.offset || '0 0';
+    var wrap = data.wrap;
+

```

Comment:
We need to add this variables to the schema of the flat and standard shaders (if applies)